etc/config/cuda.amazon.properties

```diff
@@ -56,3 +56,11 @@ libs.cucub.versions=180
 libs.cucub.url=http://nvlabs.github.io/cub/index.html
 libs.cucub.versions.180.version=1.8.0
 libs.cucub.versions.180.path=/opt/compiler-explorer/libs/cub/1.8.0
+
+libs=nsimd
+libs.nsimd.name=NSIMD
+libs.nsimd.versions=cuda
+libs.nsimd.url=https://github.com/agenium-scale/nsimd/
+libs.nsimd.versions.cuda.version=cuda
+libs.nsimd.versions.cuda.path= /opt/compiler-explorer/libs/nsimd/spmd/cuda/include
+libs.nsimd.versions.cuda=/opt/compiler-explorer/libs/nsimd/spmd/cuda/lib
```

Comment:
Super nitpick: can you ensure there's a newline at the end of the file please?

Reply:
Of course, I will add the new line right away.